app/mailers/distribution_mailer.rb

```diff
@@ -26,7 +26,13 @@ def partner_mailer(current_organization, distribution, subject, distribution_cha
     @distribution_changes = distribution_changes
     pdf = DistributionPdf.new(current_organization, @distribution).compute_and_render
     attachments[format("%s %s.pdf", @partner.name, @distribution.created_at.strftime("%Y-%m-%d"))] = pdf
-    mail(to: requestee_email, cc: @partner.email, subject: "#{subject} from #{current_organization.name}")
+    cc = [@partner.email]
+    if distribution.pick_up?
+      pickup_person_email = @partner.profile&.pick_up_email
+      cc.push(pickup_person_email) if pickup_person_email.present? && pickup_person_email != @partner.email
+    end
```

Comment:
I'd probably do it more like this:

```ruby
cc = [@partner.email]
cc.push(@partner.profile&.pick_up_email) if distribution.pick_up?
cc.compact!.uniq!
```

Reply:
i like it! Cleaner and more readable. It requires one edit:
```ruby
    cc = [@partner.email]
    cc.push(@partner.profile&.pick_up_email) if distribution.pick_up?
    cc.compact!
    cc.uniq!
```
Does it look ok for you? 
Im curious which variant works faster. However performance shouldn't be an issue here.

Reply:
addressed [48c65c2](https://github.com/rubyforgood/human-essentials/pull/4104/commits/48c65c2265653754adcb7f03d708abd2b0b0e025)